src/components/curator-actions/child-bounties/ChildBountyCard.svelte

```diff
@@ -249,38 +163,167 @@
 {/if}
 
 <style>
-	.status {
-		font-style: italic;
+	details {
+		text-align: center;
+		border-radius: 0px 0px 6px 6px;
 	}
 
-	[data-status='Active'] {
-		background-color: theme('colors.childBountyGreen');
+	summary {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		gap: 5px;
+		cursor: pointer;
+		font-size: 12px;
+		margin: -8px -8px 0px;
+		padding-top: 8px;
 	}
 
-	[data-status='PendingPayout'] {
-		background-color: theme('colors.curatorMainBackground');
+	details[open],
+	summary {
+		padding: 8px;
 	}
 
-	[data-status='CuratorProposed'] {
-		background-color: theme('colors.childBountyGray');
+	details[open] summary {
+		margin-bottom: 8px;
 	}
 
-	[data-status='Added'] {
-		background-color: theme('colors.childBountyGray');
+	.icon {
+		font-size: 18px;
+		transition: transform 0.3s ease;
 	}
 
-	[data-status='Claimed'] {
-		background-color: theme('colors.childBountyGray');
+	details[open] .icon {
+		transform: rotate(180deg);
 	}
-	[data-status='Canceled'] {
-		background-color: theme('colors.childBountyGray');
+
+	.close-child-bounty {
+		background-color: theme('colors.backgroundChildBountyCloseDown');
+		padding: 8px 8px 0;
 	}
 
-	.childContainer {
+	.child-container {
```

Comment:
I recommend sticking to camelCase for CSS classes, so that later when we have them in CSS Modules, we can use them as `styles.childContainer` and not `styles['child-container']`